tests/unit/unitTests.ml

```diff
@@ -27,6 +27,7 @@ let suites =
            Effects.suite;
            Quantifiers.suite;
            Closures.suite;
+           Variants.suite;
```

Comment:
Can we split this unit test definition into two, such that the database and non-database suites are in separate files?

Reply:
Not sure if I understand what you mean. None of these tests should interact with a database?

Reply:
So the lens tests do not use the database?

Reply:
as discussed elsewhere: you're right, and I split the unit tests into those using the database and those that don't